helm/chart-operator/Chart.yaml

```diff
@@ -1,6 +1,11 @@
 apiVersion: v1
 name: chart-operator
-description: A Helm chart for the chart-operator
-home: https://github.com/giantswarm/chart-operator
-appVersion: [[ .AppVersion ]]
 version: [[ .Version ]]
+appVersion: [[ .AppVersion ]]
+home: https://github.com/giantswarm/chart-operator
```

Comment:
What about adding icon with the GS logo as well?

Reply:
not super important for these control plane apps, also we'll have a cool auto generated icon based on the first letters of the app name coming up soon for apps that don't define an icon.

unless you wanna make something clever or cute that builds on the giant swarm logo, for identifying it and building an identity for chart operator... for now I would leave it without an icon.

Reply:
Thanks @oponder cool lets leave it then.